### PR TITLE
Donor FAQ Markup

### PR DESCRIPTION
**Pull request by debarati007**
Deliverables -

Visit the page plateletdonors.org/DonorsFAQs.php . This is the page we will be implementing here.
Create a new file called - "faqs.mustache"
Write markup for the main body of this page as shown in the image (ignore the header and the footer)

### CODE REVIEW

File: views/faqs.mustache [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<li>Donating platelets is a very safe procedure. Sterile , single use collection kits and needles are used once, just for you and then discarded.</li>
+<li>You can donate platelets 24 times a year compared to 4 times in a year for whole blood.</li>
+<li>Because it takes more time to separate and collect the platelets from the other components in your blood</li>
+  
+    </ul> </p>
+
+    <!-- include the Glitch button to show what the webpage is about and
+          to make it easier for folks to view source and remix -->
+    <div class="glitchButton" style="position:fixed;top:20px;right:20px;"></div>
+    <script src="https://button.glitch.me/button.js"></script>
```
Comment (shivamproso) [inline views/faqs.mustache]: Why is glitch button included?

File: views/faqs.mustache [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <!-- import the webpage's stylesheet -->
+    <link rel="stylesheet" href="/style.css">
+    
+    <!-- import the webpage's javascript file -->
+    <script src="/script.js" defer></script>
+  </head>  
+  <body>
```
Comment (saumitra2810) [inline views/faqs.mustache]: Let's remove everything above this line. So, the way we are going to do this is -

We will have a skeleton header file and skeleton footer file and we will include both those file in every route. In other words, we will not have to rewrite the header and footer everywhere again and again.

File: views/faqs.mustache [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <strong>How is a platelet donation different from a whole blood donation ?</strong></p>
+<p>
+  <ul>
+    
+ <li>Platelets are not collected at Blood Drives.</li> 
+<li>Donating platelets is a very safe procedure. Sterile , single use collection kits and needles are used once, just for you and then discarded.</li>
+<li>You can donate platelets 24 times a year compared to 4 times in a year for whole blood.</li>
+<li>Because it takes more time to separate and collect the platelets from the other components in your blood</li>
+  
+    </ul> </p>
```
Comment (saumitra2810) [inline views/faqs.mustache]: Same for everything below this.

File: public/stylesheets/style.css [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/* CSS files add styling rules to your content */
+
+body {
+  border: 2px solid yellow;
+  font-family: "Benton Sans", "Helvetica Neue", helvetica, arial, sans-serif;
+  margin: 2em;
+}
```
Comment (saumitra2810) [inline public/stylesheets/style.css]: Let's not apply classes to the body tag yet. We will do this with other tasks.

File: views/faqs.mustache [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <!-- import the webpage's stylesheet -->
+    <link rel="stylesheet" href="/style.css">
+    
+    <!-- import the webpage's javascript file -->
+    <script src="/script.js" defer></script>
+  </head>  
+  <body>
+    <h1>Donors FAQs</h1>
```
Comment (saumitra2810) [inline views/faqs.mustache]: Is the h1 tag looking as in the site? I think we need a container div, with a border and a margin. Take a look at the header in the attached image. 
screenshot 2018-11-08 at 7 22 45 am

**Pull request merged**